init.lua

```diff
@@ -64,203 +49,73 @@ function letters.register_letters(modname, subname, from_node, description, tile
 	}
 	def.legacy_wallmounted = false
 
+	for _, tile in ipairs(minetest.get_dir_list(letterspath .. "/textures", false)) do
+		local char, group = tile:match("_([%d%u%l])(%l)_overlay")
+		if char and group then
+			def = table.copy(def)
```

Comment:
This is prone to issues caused by read access to `def`, which was changed by the previous iteration. Please use a new variable here to always begin with the same "template" table.

Reply:
Can't believe I missed that. Thank you!